azure-quantum/azure/quantum/qiskit/job.py

```diff
@@ -71,12 +72,12 @@ def submit(self):
         self._azure_job.submit()
         return
 
-    def result(self, timeout=None):
+    def result(self, timeout=None, sampler_seed=None):
         """Return the results of the job."""
         self._azure_job.wait_until_completed(timeout_secs=timeout)
 
         success = self._azure_job.details.status == "Succeeded"
-        results = self._format_results()
+        results = self._format_results(sampler_seed=sampler_seed)
 
         return Result.from_dict(

```

Comment:
Can we cache the results of this call as a field of this object, so we don't have to download and parse the results everytime the user call `result()`?

Reply:
This is actually already the behavior of `Job.get_results` which is called under the hood by `_format_results`.